english/duck/duck.md

```diff
@@ -9,4 +9,7 @@ Wait, what was that noise?
 
 Large bats fly over your head.
 
-[Investigate](../cave/cave.md)
\ No newline at end of file
+A bit of LSD floats between your legs.
+
+[Investigate](../cave/cave.md)
+[Smash dat LSD](https://github.com/mattwoolz/create-your-own-adventure/blob/master/english/batmetal/batmetal.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.